frontend/talentsearch/src/js/components/Browse/OngoingRecruitmentSection.tsx

```diff
@@ -619,75 +644,128 @@ export const OngoingRecruitmentSection = ({
             "instructions for section with ongoing pool advertisements",
         })}
       </p>
-      <div data-h2-padding="base(x2, 0, 0, 0) p-tablet(x3, 0, 0, 0)">
+      <div data-h2-display="base(flex)">
+        <FormProvider {...methods}>
+          <form>
```

Comment:
If we are going to do this `onChange` then I would drop the `<form>` wrapper. How it is now, it would fail an automated audit for [G80: Providing a submit a submit button to initiate a change of context](https://www.w3.org/TR/WCAG20-TECHS/G80.html). 

I believe the `aria-live` area you added below would be enough to pass a manual test but having the `<form>` wrapper might get flagged in any automated audit and isn't entirely necessary so it may be better to remove it 👍

Reply:
Ah, good catch!  Fixed in [Fixes from Eric's review](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5048/commits/49988f14f6c07d06bd7000f1ceb32dc9f9d26e4a)